_docs/setup/kubernetes/helm.md

```diff
@@ -0,0 +1,76 @@
+Helm Charts expose configuration options which are currently in alpha.  The currently exposed options are explained in the
+following table:
+
+| Helm Variable               | Possible Values    | Default Value              | Purpose of Key                               |
+|-----------------------------|--------------------|----------------------------|----------------------------------------------|
+| global.auth_enabled         | true/false         | false                      | Enables TLS authentication                   |
+| global.namespace            | any Kubernetes ns  | istio-system               | Specifies the namespace for Istio            |
+| global.proxy_hub            | registry+namespace | release registry/namespace | Specifies the HUB for the proxy image        |
+| global.proxy_tag            | image tag          | release unique hash        | Specifies the TAG for the proxy image        |
+| global.proxy_debug          | true/false         | false                      | Specifies whether proxy is run in debug mode |
+| global.pilot_hub            | registry+namespace | release registry/namespace | Specifies the HUB for the pilot image        |
+| global.pilot_tag            | image tag          | release unique hash        | Specifies the TAG for the pilot image        |
+| global.mixer_hub            | registry+namespace | release registry/namespace | Specifies the HUB for the mixer image        |
+| global.mixer_tag            | image tag          | release unique hash        | Specifies the TAG for the mixer image        |
+| global.hyperkube_hub        | registry+namesapce | quay.io/coreos/hyperkube   | Specifies the HUB for the hyperkube image    |
+| global.hyperkube_tag        | image tag          | v1.7.6_coreos.0            | Specifies the TAG for the hyperkube image    |
+| istio.deploy_base_config    | true/false         | true                       | NEEDS REVIEW                                 |
+| istio.initializer_enabled   | true/false         | true                       | Specifies whether to use the Initializer     |
+| istio.mixer_enabled         | true/false         | true                       | Specifies whether mixer is enabled/disabled  |
+| istio.pilot_enabled         | true/false	   | true                       | Specifies whether pilot is enabled/disabled  |
+| istio.ca_hub:               | registry+namespace | release registry/namespace | Specifies the HUB for the ca image           |
+| istio.ca_tag:               | image tag          | release unique hash        | Specifies the TAG for the ca image           |
+| istio.ca_demo               | true/false         | false                      | NEEDS REVIEW                                 |
+| istio.ingress.use_nodeport  | true/false         | false                      | Specifies whether to use nodeport or LB      |
+| istio.ingress.nodeport_port | 1024 to 65535      | 32000                      | If nodeport is used, specifies its port      |
+
```

Comment:
could you update this?  This was changed yesterday to simplify things.

Reply:
updated.  Agree much simpler.